controllers/toolchaincluster/healthchecker_test.go

```diff
@@ -19,6 +19,7 @@ import (
 func TestClusterHealthChecks(t *testing.T) {
 	// given
 	defer gock.Off()
+	ons := "test-namespace"
```

Comment:
of `ons` stands for "operator namespace', then I'm afraid that you mix two values together. 
operator namespace is the namespace where the operator that ToolchainCluster represents is installed in - this is stored as a label 
but what you are setting here is the namespace where the ToolchainCluster is created.
so could you please just rename the variable to something more meaningful? eg:
* `tcNs`
* `toolchainClusterNs`

It's just a detail, but it may confuse people.

Reply:
Okay i agree, changed it